modules/raft/src/main/java/org/apache/ignite/internal/raft/Loza.java

```diff
@@ -149,6 +149,8 @@ public Loza(
     public void start() {
         opts.setRpcInstallSnapshotTimeout(raftConfiguration.rpcInstallSnapshotTimeout().value());
 
+        opts.getRaftOptions().setSync(raftConfiguration.fsync().value());
```

Comment:
Should we first take `RaftView` and then use it to obtain the 2 settings (to make sure that we always get a consistent snapshot of the configuration)?

Reply:
Fix it